transactions/lockedTokens/admin/unlock_tokens_for_multiple_accounts.cdc

```diff
@@ -56,7 +56,11 @@ transaction(unlockInfo: {Address: UFix64}) {
 
         admin.storage.save<{Address: UFix64}>(badAccounts, to: /storage/unlockingBadAccounts)
 
-        let unlockingBadAccountCap = admin.capabilities.storage.issue<&{Address: UFix64}>(/storage/unlockingBadAccounts)
-        admin.capabilities.publish(unlockingBadAccountCap, at: /public/unlockingBadAccounts)
+        let unlockingBadAccountCap = admin.capabilities.get<&{Address: UFix64}>(/public/unlockingBadAccounts)
+        if !unlockingBadAccountCap.check() {
+            let unlockingBadAccountCap = admin.capabilities.storage.issue<&{Address: UFix64}>(/storage/unlockingBadAccounts)
+            admin.capabilities.unpublish(/public/unlockingBadAccounts)
```

Comment:
I'm guessing unpublishing a non-existent path is just a no-op?

Reply:
correct, nothing will happen if there isn't anything there